packages/cryptoplease/test/golden/screens/home_flow/send_flow/create_link_content_test.dart

```diff
@@ -11,6 +11,7 @@ void main() {
         amount: '1.25 SOL',
         fee: '0.01 SOL',
         fiatAmount: '123.45 USD',
+        fiatFee: '0.0001 USD',
```

Comment:
This looks wrong. `0.01` is the minimal possible amount for USD. For a smaller fee, it makes sense to format it as `< 0.01 USD`.